packages/frontend/editor-ui/src/features/ai/chatHub/components/ChatPrompt.vue

```diff
@@ -109,20 +113,56 @@ function handleKeydownTextarea(e: KeyboardEvent) {
 		speechInput.stop();
 		emit('submit', trimmed, attachments.value);
 		message.value = '';
+		committedSpokenMessage.value = '';
 		attachments.value = [];
 	}
 }
 
+function handlePaste(e: ClipboardEvent) {
+	e.preventDefault();
+
+	const text = e.clipboardData?.getData('text');
+
+	if (!text) {
+		return;
+	}
+
+	// Text copied from conversation history tends to
+	// include unnecessary leading & trailing whitespaces, so we're removing them
```

Comment:
I didn't really get what the use case of pasting from conversation history to chat again really is, but we can of course try to prevent this on paste but normally I would expect the user to be pasting the output elsewhere, not back to the same browser 🤔

Reply:
True. Reverted paste event handling and implemented a fix in ChatMessage itself [04427c9](https://github.com/n8n-io/n8n/pull/22109/commits/04427c99e6b78635b25795f845f6ba591b216d3b)